clientapi/auth/user_interactive.go

```diff
@@ -157,10 +156,11 @@ type Challenge struct {
 
 // Challenge returns an HTTP 401 with the supported flows for authenticating
 func (u *UserInteractive) challenge(sessionID string) *util.JSONResponse {
-	u.RLock()
-	completed := u.Sessions[sessionID]
-	flows := u.Flows
-	u.RUnlock()
+	u.mutex.RLock()
+	completed := u.sessions[sessionID]
+	u.mutex.RUnlock()
+
+	flows := u.flows
```

Comment:
Minor nit: No need to create `flows` when we can use `u.flows` below directly.

Reply:
- [x] using `u.flows` directly